test/nextflow.config

```diff
@@ -5,6 +5,8 @@
 params {
     mode = "run"
 
+    read_type = "paired_end"
```

Comment:
As above, we shouldn't need this

Reply:
You say implemented, but this pull request still shows you defining read type, am I reviewing the wrong commits?